### ISSUE TITLE
Totally missing documentation for styling

### ISSUE THREAD

**Issue by bmarkovic** [shentao/vue-multiselect#1214]
### Reproduction Link

https://vue-multiselect.js.org/ 

### Steps to reproduce

Read the documentation.

### Expected behaviour

At least a basic documentation on styling. 

### Actual behaviour

The compnent is complex and consitsts of multiple styleable elements, figuring out what is what is nontrivial, and even basic styling turns out to be extremely irritating endless trial and error.

**Comment by tfdbloom** [comment]
I agree. The component itself is great but there is not much mention of styling in the documentation, if some? I was somewhat expecting a section on style, with the element structure and classes, with more details on how to style. A "debug mode" would nice also to keep it open during styling, as it is now it is a bit frustrating imo.

**Comment by synapse709** [comment]
Agreed, I'm looking through the docs now and shocked that there isn't a list of classes that outlines the component. Would really like to see this covered in detail (some of us have to match style guides!)
